hyp3_metadata/templates/dem/dem-cop.xml.j2

```diff
@@ -3,9 +3,9 @@
 a global Digital Surface Model (DSM) at 30-m pixel spacing derived from the WorldDEM. The WorldDEM is based on radar satellite data acquired by the TanDEM-X mission, which was funded by the German Aerospace Center (DLR) and Airbus Defence and Space, and edited to flatten water bodies, provide consistent flow of rivers, and apply corrections to shore/coastlines and special features. For more information, refer to &lt;/SPAN&gt;&lt;A href="https://spacedata.copernicus.eu/documents/20126/0/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I1.00.pdf" STYLE="text-decoration:underline;"&gt;&lt;SPAN&gt;https://spacedata.copernicus.eu/documents/20126/0/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I1.00.pdf&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;The DEM data used as input are distributed by Sinergise, and are available for download at &lt;/SPAN&gt;&lt;A href="https://registry.opendata.aws:443/copernicus-dem/" STYLE="text-decoration:underline;"&gt;&lt;SPAN&gt;https://registry.opendata.aws/copernicus-dem/&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;. To prepare the DEM for use in RTC processing, the tiles necessary for coverage of the SAR granule are downloaded, and a mosaic is generated and reprojected to the appropriate UTM Zone and pixel spacing. The GAMMA dem_import utility is used to apply a geoid correction. This DEM is a 16-bit signed integer version of the 32-bit float DEM used for processing. &lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;The actual {{ scale }} data is contained in the VV/HH (primary polarization) and/or VH/HV (cross-polarization) TIFF files included in the same downloaded folder as this file. The name of the SAR granule used to generate the files in this folder is: {{ granule_name }}. For a detailed description of the Sentinel-1 file naming convention, refer to &lt;/SPAN&gt;&lt;A href="https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/sentinel-1-data-and-imagery/#Naming_Convention"&gt;&lt;SPAN&gt;https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/sentinel-1-data-and-imagery/#Naming_Convention&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;The Sentinel-1A satellite was launched April 3, 2014, and the Sentinel-1B satellite was launched April 25, 2016. The satellites each have a 12-day repeat cycle. More information about the mission is available at &lt;/SPAN&gt;&lt;A href="https://earth.esa.int/web/guest/missions/esa-operational-eo-missions/sentinel-1"&gt;&lt;SPAN&gt;https://earth.esa.int/web/guest/missions/esa-operational-eo-missions/sentinel-1&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Additional information about Sentinel-1 data, imagery, tools and applications is available at &lt;/SPAN&gt;&lt;A href="https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/"&gt;&lt;SPAN&gt;https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;&lt;/DIV&gt;
```

Comment:
The details of preparing the DEM are included twice, we might not need both.  I like the second description better.

`and the extent of the DEM is clipped to the size needed for full granule coverage, or to the extent of the available DEM data if full coverage is not available. The DEM was reprojected to {{ projection }}, and the pixel spacing was resampled from the native DEM pixel spacing to {{ pixel_spacing|int }} m`

`the tiles necessary for coverage of the SAR granule are downloaded, and a mosaic is generated and reprojected to the appropriate UTM Zone and pixel spacing. The GAMMA dem_import utility is used to apply a geoid correction.`

Reply:
I've streamlined the description, and cleaned up a bunch of other random bits and pieces.